test/Main.purs

```diff
@@ -23,10 +21,10 @@ newtype TestRational = TestRational Rational
 derive newtype instance commutativeRingTestRational :: CommutativeRing TestRational
 derive newtype instance eqTestRational :: Eq TestRational
 derive newtype instance euclideanRingTestRational :: EuclideanRing TestRational
-derive newtype instance fieldTestRational :: Field TestRational
 derive newtype instance ordTestRational :: Ord TestRational
 derive newtype instance ringTestRational :: Ring TestRational
 derive newtype instance semiringTestRational :: Semiring TestRational
+derive newtype instance divisionRingTestRational :: DivisionRing TestRational
```

Comment:
You can see that used here. But as you'll note there isn't any change to the test suite. If I need to add division-specific testing for this purpose, I can take a look at that @anttih

Reply:
I'd like to have those tests yes. I saw some discussion of switching to quickcheck, I'd be ok with that if that makes things easier.

Reply:
It looks like there isn't a particular law in `quickcheck-laws` or `strongcheck-laws` that would be necessary to cover the division ring, and I *believe* that the `Field` tests would already include testing for division as all fields are by definition division rings.

However, I'll ask a few folks with a better understanding of the nuances here for verification.

With regards to StrongCheck vs. QuickCheck -- it's simply a matter of how quickly the tests run, and for the test suite here I don't think it matters which one you use.

Reply:
Let's go with this for now at least. Can you squash and I'll merge?

Reply:
👍 done.